data/json/items/migration.json

```diff
@@ -149,5 +149,11 @@
     "type": "MIGRATION",
     "replace": "jar_glass_sealed",
     "contents": [ "tomato_canned" ]
+  },
+  {
+    "id": "jar_kompot",
```

Comment:
That's a pretty minor detail that's rather tedious to keep up with manually.

Is there a tool that can do it automatically?